exercises/concept/lasagna-master/.docs/hints.md

```diff
@@ -20,5 +20,7 @@
 - First make a new slice of the same size as the input slice
 - Use a [for range loop][concept-range-iteration] to iterate through the input slice and generate the output slice
 
+[len-builtin]: https://pkg.go.dev/builtin#len
 [concept-conditonals-if]: /tracks/go/concepts/conditionals-if
 [concept-slices]: /tracks/go/concepts/slices
+[concept-range-iteration]: tracks/go/concepts/range-iteration
```

Comment:
at this point in the learning curve range iteration is not covered yet (it needs maps which will  need functions in the future).

Could you make sure all hints refer to normal for loops instead with some external link that explains those? the internal for loops concept is not live

Reply:
Done!